tools/assets_generate/main.go

```diff
@@ -18,7 +18,7 @@ import (
 	"net/http"
 	"os"
 
-	"github.com/shurcooL/vfsgen"
+	"github.com/cwen0/vfsgen"
```

Comment:
Is this change intended? You'd like to use own fork here?

Reply:
The origin repo doesn't support setting the fallback and the repo is rarely updated, but we need this feature. https://github.com/cwen0/vfsgen/commit/c35322782328a576b31b802094d540f346ac0b39

Reply:
I will try to push this pr to the origin repo.